tools/typescript.js

```diff
@@ -1,4 +1,3 @@
-//typescript v 1.6.0-beta

```

Comment:
can you keep + update the version number? otherwise we won't know what is being used.